terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -231,8 +231,11 @@
 
 other_public_keys = {
     'dev': operator_keys,
-    'anvildev': operator_keys,
-    'anvilprod': operator_keys,
+    'anvildev': operator_keys
+    # Operators are granted ssh access only to the non-stable GitLab instances
+    # in order to prevent any unforced erros in the stable deployments.
```

Comment:
```suggestion
    # in order to prevent any unforced errors in the stable deployments.
```

😆 
However I'm not sure if the comment is needed. Especially if each deployment is checked with `is_stable` like:
```
other_public_keys = {
    name: [] if config.Deployment(name).is_stable else operator_keys
    for name in ('dev', 'anvildev', 'anvilprod', 'prod')
}
```